tools/manifest/item.py

```diff
@@ -46,14 +46,19 @@ def id(self):
         """The test's id (usually its url)"""
         pass
 
+    @property
+    def meta_flags(self):
+        return self.source_file.meta_flags
```

Comment:
Should this logically not be a set rather than a list? (I realise it currently isn't, and I'm not gonna block this PR on that.)